host/bcm283x/gpio_test.go

```diff
@@ -160,6 +162,7 @@ func TestPinPWM(t *testing.T) {
 		t.Fatal(err)
 	}
 	// TODO(maruel): Fix test.
+	dmaMemory = &dmaMap{}
```

Comment:
defer func() { dmaMemory = nil }()
so the value is reset correctly.

Reply:
done.